packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -187,16 +188,51 @@ export class AppStudioPluginImpl {
       if (solutionSettings.capabilities.includes(MessageExtensionItem.id)) {
         manifest.composeExtensions = COMPOSE_EXTENSIONS_TPL;
       }
+
+      if (settings?.solutionSettings?.migrateFromV1) {
```

Comment:
How is it possible a migrated project calls `createManifest`? Since appstudio plugin is inactive for migrated project.

Reply:
App studio plugin, local debug plugin and frontend hosting plugin will be active for a migrated project.

Reply:
For the v1 project created before v1.2.0, there is no manifest.json file in the project. so we will the same logic to create a new one.